lib/src/id_prefix.rs

```diff
@@ -136,6 +238,8 @@ where
             })
             .max()
             .unwrap_or(0)
+            // Even if the key is the only one in the index, we require at least one digit.
+            .max(1)
```

Comment:
`.unwrap_or(1)` ?

If the index only contained the `key`, both `left` and `right` would be `None`.